src/BaseFirestoreRepository.spec.ts

```diff
@@ -237,6 +284,55 @@ describe('BaseFirestoreRepository', () => {
       const updatedBand = await bandRepository.update(band);
       expect(band.name).to.equal(updatedBand.name);
     });
+
+    it('must not validate if the validate config property is false', async () => {
+      initialize(firestore, { validate: false });
+
+      bandRepository = new BandRepository('bands');
+
+      const band = await bandRepository.findById('porcupine-tree');
+
+      band.contactEmail = 'test@email.com';
+
+      await expect(bandRepository.update(band)).not.to.be.rejected;
+    });
+
+    it('must pass validation if a valid class is given', async () => {
+      const band = await bandRepository.findById('porcupine-tree');
+
+      band.contactEmail = 'test@email.com';
+
+      await expect(bandRepository.update(band)).not.to.be.rejected;
+    });
+
+    it('must pass validation if a valid object is given', async () => {
+      const band = await bandRepository.findById('porcupine-tree');
+      const updatedBand: Partial<Band> = {
+        ...band,
+        contactEmail: 'test@email.com',
+      }
+
+      await expect(bandRepository.update(updatedBand as Band)).not.to.be.rejected;
+    });
+
+    it('must fail validation if an invalid class is given', async () => {
```

Comment:
A missing unit test is: `must pass if it has validation errors but runValidations is false`

Reply:
Shouldn't be needed once the other test uses an invalid value

Reply:
Yes, saw the wrong test afterwards and forgot to delete this comment 😄